Library/Formula/julius.rb

```diff
@@ -0,0 +1,18 @@
+class Julius < Formula
+  homepage "http://julius.sourceforge.jp"
+  url "http://sourceforge.jp/frs/redir.php?m=iij&f=%2Fjulius%2F60273%2Fjulius-4.3.1.tar.gz"
+  sha1 "88f64ae9ed00b6ab5a2d4fe07e3ced141a46c196"
+
+  depends_on "libsndfile"
+
+  def install
+
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--mandir=#{man}",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!